handlers.py

```diff
@@ -34,7 +35,20 @@ def book_earliest_date(driver):
     all_tee_times[0].click()
 
 
-def continue_to_book(driver):
+def continue_to_book(driver, golfers):
+    num_golfers_buttons = WebDriverWait(driver, 10).until(
+        EC.presence_of_all_elements_located(
+            (
+                By.CSS_SELECTOR,
+                '[data-testid="player-count"] > button'
+            )
+        )
+    )
+    if int(golfers) > len(num_golfers_buttons):
+        num_golfers_buttons[-1].click()
+    else:
+        num_golfers_buttons[int(golfers)-1].click()
```

Comment:
This doesn't seem right to me. Number of golfers will be a number between 1 and 4. Since there are 4 buttons, this `if` will never trigger.

I think this should just be:
```suggestion
num_golfers_buttons[int(golfers)-1].click()
```
x {1,4} into function  y = x - 1 maps y to {0,3} (index of the button in the list), which seems correct but I could be wrong.

Reply:
its more of a safety  and sanity check. when you initially click book on a tee time, in the time it takes to go to the next page someone could have taken one of the spots. if we always did `num_golfers_buttons[int(golfers)-1].click()` then it could cause index out of bounds error

example:
- `--num-golfers=4`
- click on tee time that has 4 spots available
- when on create booking screen, one spot was already taken so there are 3 buttons
- `len(num_golfers_buttons)` = 3
- `num_golfers_buttons[int(golfers)-1]` -> `num_golfers_buttons[3]` which wouldn't exist

Reply:
ah okay, wasn't aware the number of buttons on that page changed dynamically like that. this is fine then.